README.md

```diff
@@ -401,13 +401,13 @@ module.exports = {
 
 Webpack plugin will also land a "hot" patch to react-dom, making React-Hot-Loader more compliant to [the principles](https://github.com/gaearon/react-hot-loader/issues/1118).
 
-If you are using `babel-plugin` you might not need to apply `webpack-loader` to all the files, scoping it to `react-dom`
+If you are not using `babel` plugin you might need to apply `webpack-loader` to all the files.
 
 ```js
 // would only land a "hot-patch" to react-dom
```

Comment:
this comment should be moved to the code example above.

Reply:
Yep my bad! Thanks, fixed.